semgrep_output_v1.atd

```diff
@@ -898,6 +894,29 @@ type dependency_parser_error = {
 (* CI Configuration *)
 (* ----------------------------- *)
 
+(* Response by the backend to the CLI to the POST deployments/current *)
+type deployment_config <ocaml attr="deriving show"> = {
```

Comment:
Nit picky, but "config" is the wrong word to use here IMO - config implies something else, and I'm trying to split these concepts up in a separate PR.

My preference would be to not type this endpoint at all since it's not dedicated to the CLI and therefore will never be typed with ATD in the backend. 

In a separate PR that I'm working on backend changes for I actually stop calling this endpoint anyway.

Reply:
I'll be happy to remove this type from the ATD, but right now it's still used in osemgrep so I need it.